documentation/cli/docs/03_account.md

```diff
@@ -396,34 +396,20 @@ Sends an amount to an address.
 
 #### Parameters
 
-| Name      | Optional  | Default | Example                                                         |
-| --------- | --------- | ------- | --------------------------------------------------------------- |
-| `address` | ✘         | N/A     | rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 |
-| `amount`  | ✘         | N/A     | 1000000                                                         |
+| Name                 | Optional  | Default | Example                                                         |
+| -------------------- | --------- | ------- | --------------------------------------------------------------- |
+| `address`            | ✘         | N/A     | rms1qztwng6cty8cfm42nzvq099ev7udhrnk0rw8jt8vttf9kpqnxhpsx869vr3 |
+| `amount`             | ✘         | N/A     | 1000000                                                         |
+| `return_address`     | ✓         | 1st     | smr1qrvddncqclzrxrkxhuy4cjr9aqth5nu0h5kuzhr4phj5u67mmq29zmywk33 |
+| `expiration`         | ✓         | 1d      | 72h                                                             |
+| `allow_micro_amount` | ✓         | false   | N/A                                                             |
```

Comment:
Maybe put true as example, N/A may confuse people.
I would have done a suggestion but it's a bit annoying to get right with the table and spaces

Reply:
You don't actually set a value. It's used like this `send --allow-micro-amount`. You still think it should be true?

Reply:
Ah it's a flag, no worries then